src/invoice_utils/config.py

```diff
@@ -28,3 +29,8 @@ def _str_to_bool(arg: str) -> bool:
     "INVOICE_UTILS_TEMPLATES_DIR",
     DEFAULT_BODY_TEMPLATE_DIRECTORY
 )
+INVOICE_UTILS_DEFAULT_RULE_TEMPLATE_NAME = os.getenv(
```

Comment:
This variable should be called `INVOICE_UTILS_RULE_TEMPLATE_NAME`. The `DEFAULT_...` is specified as a constant above.